website/content/docs/services/discovery/dns-forwarding/enable.mdx

```diff
@@ -0,0 +1,442 @@
+If Consul is not listening on the loopback IP, replace the references to `localhost` and `120.0.0.1` in the configuration and commands with the appropriate IP address for your environment.
+
+</Note>
+
+## systemm-resolved
+
+[`systemd-resolved`](https://www.freedesktop.org/wiki/Software/systemd/resolved/) is a system service that provides network name resolution to local applications. Is the default local DNS server for many Linux distributions.
+
+You can configure the `systemd-resolved` service to send `.consul` domain queries to Consul by creating a `consul.conf` file located in the `/etc/systemd/resolved.conf.d/` directory.
+
+<Tabs>
+<Tab heading="systemd 245 and older">
+
+Add the following settings to your resolved configuration file:
+
+<CodeBlockConfig filename="/etc/systemd/resolved.conf.d/consul.conf">
+
+```ini
+[Resolve]
+DNS=127.0.0.1
+DNSSEC=false
+Domains=~consul
+```
+
+</CodeBlockConfig>
+
+### Define port for Consul DNS server
+
+When using systemd 245 and older, you cannot specify port numbers in the `DNS` configuration field. Resolved will only use port `53`, which is a privileged port.
+
+The two possible solutions for this limitation are:
+ - [configure Consul DNS service to listen on port `53`](/consul/docs/agent/config/config-files#dns_port) instead of `8600`
+ - map port `53` to `8600` using `iptables` 
+
+Binding to port `53` usually requires running Consul as a privileged user or running Linux with the `CAP_NET_BIND_SERVICE` capability. 
+When using Consul Docker image, add the following to the environment to allow Consul to use the port: `CONSUL_ALLOW_PRIVILEGED_PORTS=yes`.
+
+To avoid running Consul as a privileged user, the following `iptables` commands are sufficient to map port `53` to `8600`, redirecting DNS queries to Consul.
+
+```shell-session
+# iptables --table nat --append OUTPUT --destination localhost --protocol udp --match udp --dport 53 --jump REDIRECT --to-ports 8600 \
+  iptables --table nat --append OUTPUT --destination localhost --protocol tcp --match tcp --dport 53 --jump REDIRECT --to-ports 8600
+```
+
+</Tab>
+<Tab heading="systemd 246 and newer">
+
+Systemd 246 and newer allow you to specify the DNS port directly in the `systemd-resolved` configuration file.
+Previous versions of systemd required iptables rules to direct DNS traffic to Consul.
+
+Add the following settings to your resolved configuration file:
+
+<CodeBlockConfig filename="/etc/systemd/resolved.conf.d/consul.conf">
+
+```ini
+[Resolve]
+DNS=127.0.0.1:8600
+DNSSEC=false
+Domains=~consul
+```
+
+</CodeBlockConfig>
+
+</Tab>
+</Tabs>
+
+<Note>
+
+PTR record queries will still be sent out to the other configured resolvers, in addition to Consul.
+
+</Note>
+
+After creating the resolved configuration, restart `systemd-resolved`.
+
+```shell-session
+# systemctl restart systemd-resolved
+```
+
+The command produces no output.
+
+### Validate the systemd-resolved configuration
+
```

Comment:
@aimeeu and @boruszak  the commands are not mandatory to complete the configuration but only to verify that the configuration is correctly performed. They are in logical order but, since none of them is changing anything in the configuration, they will work even if performed in a different order.

The order provided is only to move from the wider context to a more narrow scope.

First 3 are to check that the DNS configuration is in place:
* DNS server is up and running
* DNS server is configured to forward to Consul
* DNS server is able to resolve *.consul queries

If not, we need to go back to the DNS server config and check what is missing

Second 3 are to check if OS configuration is in place:
* OS /etc/resolv.conf file points to the correct stub generated by the DNS server
* OS /etc/resolv.conf content points to Consul DNS
* OS is able to resolve *.consul queries

If not we need to investigate OS configuration.